modules/programs/vim.nix

```diff
@@ -75,8 +75,8 @@ in
         default = defaultPlugins;
         example = [ "YankRing" ];
         description = ''
-          List of vim plugins to install. For supported plugins see:
-          <link xlink:href="https://github.com/NixOS/nixpkgs/blob/master/pkgs/misc/vim-plugins/vim-plugin-names"/>.
+          List of vim plugins to install. To get a list of supported plugins run:
+          <command>$ nix-env -f '<nixpkgs>' -qaP -A vimPlugins --description</command>
```

Comment:
Can remove `$ ` and possibly also ` --description` because only one plugin seems to have a description anyway:
```console
$ diff <(nix-env -f '<nixpkgs>' -qaP -A vimPlugins --description | sed 's/ *$//') <(nix-env -f '<nixpkgs>' -qaP -A vimPlugins)
276c276
< vimPlugins.YouCompleteMe                   vimplugin-youcompleteme-2018-03-09                   Fastest non utf-8 aware word and C completion engine for Vim
---
> vimPlugins.YouCompleteMe                   vimplugin-youcompleteme-2018-03-09
```
i.e., to have
```xml
<command>nix-env -f '<nixpkgs>' -qaP -A vimPlugins</command>
```